src/facility.ts

```diff
@@ -220,7 +220,7 @@ const fac: Facility = {
   // This merges with the links in src/facility.ts using title as the key
   links: [
     {
-      title: "Facility",
+      title: "Profile",
```

Comment:
This should be in `facility-zdv.ts` as facility.ts is overridden during the build process.